micro_dl/input/dataset.py

```diff
@@ -69,21 +93,35 @@ def _augment_image(self, input_image, aug_idx):
          5 - rot 270
         :return np.array image after transformation is applied
         """
+        # We need to flip over different dimensions depending on data format
+        add_dim = 0
+        if self.data_format == 'channels_first':
```

Comment:
may be: ```if len(input_image.shape)==3: ```

Reply:
Good ideay, but I think I changed the tiles to always have a singleton dimension, so len is always 3...